spec/integration_spec.rb

```diff
@@ -99,7 +158,7 @@ def run_tests(test_option: nil)
         type: 'rspec',
         processes: 2,
         # group-by + order for stable execution ... doc and verbose to ease debugging
-        add: "--group-by found --verbose --fail-fast --test-options '--format doc --order defined #{test_option}'"
+        add:  "--group-by found --verbose --fail-fast --test-options '--format doc --order defined #{test_option}'"
```

Comment:
```suggestion
        add: "--group-by found --verbose --fail-fast --test-options '--format doc --order defined #{test_option}'"
```

Reply:
Whoops, that wasn't meant to be in there. Removed in 3a62cb0